package.json

```diff
@@ -16,10 +19,11 @@
     "clean": "gts clean",
     "codecov": "c8 report --reporter=json && codecov -f coverage/*.json",
     "fix": "gts fix",
-    "compile": "tsc -p .",
+    "compile": "tsc -p tsconfig.compile.json",
```

Comment:
Having a `compile` and a `build` command feels wildly confusing to me.  Can you describe how these are different, and in which situations they're used?

Reply:
The build produces 3 formats, and I think these aren't necessary for the tests.
For this reason I kept them separated, but I agree it's not really DRY 🤔

Reply:
I think I'm ok with just changing the target for TypeScript to `es2015` if that helps keep this simpler?  Additionally, maybe `build-web` would be a better name?